packages/grid/src/docs/Component.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { ComponentHeader, CssVars, TabContainer } from 'storybook/blocks';
+
+import { Row, Column } from '../';
+import { version } from '../../package.json';
+import Description from './Description.mdx';
+import Props from './Props.mdx';
+import Changelog from '../../CHANGELOG.md';
+
+export const VIEWS = ['primary', 'secondary'];
+export const SIZES = ['small', 'middle', 'large'];
+
+<Meta
+  title="Components/Grid"
+  argTypes={{
+      justifyContent: {
+        control: { type: 'select', options: ['start', 'end', 'center', 'space-between', 'space-evenly', 'space-around'] }
+      },
+      alignItems: {
+        control: { type: 'select', options: ['start', 'end', 'center', 'stretch', 'baseline'] }
+      }
+    }}
+/>
+
+export const GridTemplate = args => (
+    <Row {...args}>
+        <Column widths={[12, 8, 6]}>
+             <div style={{ padding: 8, color: 'white', background: 'red', height: '100%' }}>
+                SM-12 MD-8 LG-6<br />
+                First column
+             </div>
+        </Column>
+        <Column widths={[12, 4, 6]}>
+            <div style={{ padding: 8, color: 'white', background: 'blue', height: '100%' }}>SM-12 MD-4 LG-6</div>
+        </Column>
+    </Row>
+);
```

Comment:
it looks like here not work the code formatter.
too large indent from the left side.